.vib/grafana-mimir/cypress/cypress/integration/grafana_mimir_spec.js

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+import body from '../fixtures/metric-body.json';
+import {
+  lastMinuteTimestamp,
+  random
+} from '../support/utils';
+
+it('can push metrics', () => {
+  // Using Open Telemetry endpoint because it allow us to use plain connection.
+  // Regular prometheus endpoint requires snappy compression.
+  const upToDateJson = JSON.stringify(body).replace(
+    '"timestamp_placeholder"',
+    lastMinuteTimestamp
+  ).replace(
+    '"integer_placeholder"',
+    random
+  );
+  cy.request({
+    method: 'POST',
+    headers: { 'X-Scope-OrgID': 'demo' },
+    url: '/otlp/v1/metrics',
+    body: JSON.parse(upToDateJson),
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+  });
+});
+
+it('can read metrics', () => {
```

Comment:
As stated in the [Testing guidelines](https://github.com/bitnami/charts/blob/main/TESTING.md#generic-acceptance-criteria), tests should be independent and stateless. That means that tests should not rely on the output of a previous test.

Merge both tests cases into one --> `can push metrics and read them`